packages/firebase_crashlytics/test/firebase_crashlytics_test.dart

```diff
@@ -32,10 +33,7 @@ void main() {
         exception: 'foo exception',
         stack: StackTrace.current,
         library: 'foo library',
-        context: 'foo context',
-        informationCollector: (StringBuffer information) {
-          information.writeln('foo information');
-        },
+        context: DiagnosticsNode.message('foo context'),
```

Comment:
nit: use ErrorDescription('foo context')
as that is the idiomatic version to use for error collection.